### ISSUE TITLE
[Xamarin.Forms][Idea] Merging multiple images into a single one

### ISSUE THREAD

**Issue by toussam** [luberda-molinet/FFImageLoading#956]
Hi Daniel,

I have been using your library for a while now and it's working great ! Thank you for everything !!

As I said into [this topic](https://forums.xamarin.com/discussion/123144/is-there-any-solution-to-merge-multiple-images-to-create-a-groups-picture#latest), I am looking for a way to merge multiple images into a single one.

### Idea

Merging multiple images to create a single and unique picture that will display some or all of them into a mosaic style. This will be quite useful to illustrate a group a bit like Messenger is doing for the group chat.

### Links

[iOS' recipe to merge images](https://developer.xamarin.com/recipes/ios/media/images/merge_images/)

Do you think that it could be a good feature for the FFImageLoading library ? :)

Thank you in advance for your response !
Best regards,

**Comment by daniel-luberda** [comment]
Yes, it would be nice to add that kind of transformation, eg. `MergeTransformation` would be nice. 

Some other sources: 
- Windows implementation: https://github.com/teichgraf/WriteableBitmapEx/blob/master/Source/WriteableBitmapEx/WriteableBitmapBlitExtensions.cs
- Android implementation: https://stackoverflow.com/questions/31813638/how-to-merge-bitmaps-in-android

Any PR's are welcome, even partially working - I'll try to complete it.

**Comment by toussam** [comment]
Great ! Since this feature is not a priority for the project I am currently working on, I can not help you developing it. The day it will, if the enhancement has not been finished then I will give you some help to implement it. :)
